src/constant/index.ts

```diff
@@ -317,6 +320,10 @@ export const HARDWARE_KEYRING_TYPES = {
     type: 'imKey Hardware',
     brandName: 'imKey',
   },
+  NgraveZero: {
+    type: 'QR Hardware Wallet Device',
+    brandName: 'NGRAVE ZERO',
+  },
```

Comment:
```suggestion
```

Reply:
There is no need to set `HARDWARE_KEYRING_TYPES`, because no new keyring is added